pkgs/development/ocaml-modules/rope/default.nix

```diff
@@ -3,9 +3,9 @@
 let param =
   if stdenv.lib.versionAtLeast ocaml.version "4.03"
   then {
-    version = "0.6.1";
-    url = " https://github.com/Chris00/ocaml-rope/releases/download/0.6.1/rope-0.6.1.tbz";
-    sha256 = "1zqh28jz1zjb0l354wi1046qpkwmk582ssz0gsqh6d44wpspdxk2";
+    version = "0.6.2";
+    url = " https://github.com/Chris00/ocaml-rope/releases/download/0.6.2/rope-0.6.2.tbz";
```

Comment:
Leading space in URL? Also, would it be easier to use `rec` and splice in `$version` ?